hkmc2/shared/src/test/mlscript/handlers/EffectInHandler.mls

```diff
@@ -17,11 +18,12 @@ handle h = PrintEffect with
     h.p(x)
 h.aux(3)
 //│ ╔══[ERROR] Name not found: h
-//│ ║  l.15: 	    h.p(x)
+//│ ║  l.16: 	    h.p(x)
 //│ ╙──      	    ^
 //│ ╔══[ERROR] Name not found: h
-//│ ║  l.17: 	    h.p(x)
+//│ ║  l.18: 	    h.p(x)
 //│ ╙──      	    ^
+//│ ═══[RUNTIME ERROR] Error: MLscript call unexpectedly returned `undefined`, the forbidden value.
 
```

Comment:
This diff is due to safeCall is not emitted for `cur.handlerFun(..., ...)` for some reason. Seems it is emitted for one param but not 2 params

Reply:
Note: this diff is gone due to reverting to calling handlerFun with one param during refactor.